tests/codegen/dont-shuffle-bswaps.rs

```diff
@@ -0,0 +1,36 @@
+//@ revisions: OPT2 OPT3
+//@[OPT2] compile-flags: -Copt-level=2
```

Comment:
It's a bit odd to not have OPT2 filecheck comments, but I guess the idea here is to check for the lack of a shufflevector at level 2, and check that we get the slick bswap codegen at level 3?

Reply:
yes, we're checking for "no shufflevector plz" at both levels and then for the bswap at OPT3

Reply:
for OPT3 the only way it could have a shufflevector instruction is if it smuggled itself in right between `start:` and the actual checked lines, but, idk, might as well?